akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -368,6 +368,8 @@ private[akka] class LocalActorRef private[akka] (
    */
   def getSingleChild(name: String): InternalActorRef = actorCell.getSingleChild(name)
 
+  lazy val log = Logging(_system, getClass)
```

Comment:
not sure we should add that, it grows the ActorCell size

Reply:
Right, my bad, remainder from debugging the function refs